README.md

```diff
@@ -1,8 +1,15 @@
-# ingress-node-firewall
+<p align="center">
+  <img src="logo.jpg" alt="logo" width="25%"/>
+</p>
+
+# Ingress Node Firewall
 This is the Ingress node Firewall Operator, implementing [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) for deploying Ingress node firewall daemon on kubernetes cluster.
 It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/)
 which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster
```

Comment:
```suggestion
which provides a reconcile function responsible for synchronizing resources until the desired state is reached on the cluster
```

Reply:
Can you make this fix too? Id say run the whole doc through a spell checker as well.

Reply:
I ran spell check for the following
```bash
aspell check README.md 
aspell check test/README.md
```